cmd/tracee-rules/main.go

```diff
@@ -194,13 +195,20 @@ func listSigs(w io.Writer, sigs []types.Signature) error {
 }
 
 func listEvents(w io.Writer, sigs []types.Signature) {
-	var events []string
+	m := map[string]string{}
 	for _, sig := range sigs {
 		es, _ := sig.GetSelectedEvents()
 		for _, e := range es {
-			events = append(events, e.Name)
+			m[e.Name] = e.Name
```

Comment:
Hi, thank you for reporting! 
Why just not mapping between event name and empty struct? If element exists in map then continue.

Reply:
Done, thanks @mtcherni95